test/functional/update/update-manifest-delta.bats

```diff
@@ -84,5 +84,24 @@ test_setup() {
 	)
 	assert_is_output "$expected_output"
 
+}
+
+@test "UPD063: Error when manifest delta is invalid" {
+
+	sudo mv "$WEBDIR"/20/Manifest-test-bundle1-delta-from-10 "$WEBDIR"/30/Manifest-test-bundle2-delta-from-10
```

Comment:
If I get this correctly, you are swapping those deltas so they cause the final manifest to be invalid... wasn't swupd supposed to just use the full manifests then and continue with the update?

Reply:
The manifests were deleted on test_setup

Reply:
ah, I missed that 😊